modyn/selector/internal/selector_strategies/downsampling_strategies/kcentergreedy_downsampling_strategy.py

```diff
@@ -1,16 +1,23 @@
+from modyn.config.schema.downsampling_config import KcenterGreedyDownsamplingConfig
 from modyn.selector.internal.selector_strategies.downsampling_strategies import AbstractDownsamplingStrategy
 from modyn.utils import DownsamplingMode
 
 
 class KcenterGreedyDownsamplingStrategy(AbstractDownsamplingStrategy):
-    def __init__(self, downsampling_config: dict, modyn_config: dict, pipeline_id: int, maximum_keys_in_memory: int):
+    def __init__(
+        self,
+        downsampling_config: KcenterGreedyDownsamplingConfig,
+        modyn_config: dict,
```

Comment:
How much effort would it be to also use the pydantic model here?

Reply:
It would be another PR touching 30~ files :) I will leave it as as for now